core/src/main/resources/org/apache/spark/ui/static/spark-dag-viz.js

```diff
@@ -496,7 +496,7 @@ function connectRDDs(fromRDDId, toRDDId, edgesContainer, svgContainer) {
  * Replace `/n` with `<br/>`
  */
 function replaceLineBreak(str) {
-    return str.replace("\\n", "<br/>");
+    return str.replace(/\\n/g, "<br/>");
```

Comment:
We need this because of one more "\n" added in https://github.com/apache/spark/pull/25319/files#diff-61a3c327c6093db76e3c77280ff71ae5R233?

Reply:
Yes, this is the reason